### ISSUE TITLE
The shortcut for subscript creates "e" instead of subscript.

### ISSUE THREAD

**Issue by piiskop** [ONLYOFFICE/DocumentServer#1570]
**Do you want to request a *feature* or report a *bug*?**
*bug*

**What is the current behavior?**
The shortcut for subscript creates "e" instead of subscript.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Please use the shortcut `C + .` See that _e_ appears! You can write whatever after it, hit `Enter` and just the characters appear you typed, normally, not as subscript.

**What is the expected behavior?**
Using the shortcut for subscript really creates subscript.

**Did this work in previous versions of DocumentServer?**
Yes.

**DocumentServer version:**
6.4.1.45

**Operating System:**
`lsb_release -a`

> No LSB modules are available.
> Distributor ID:	Ubuntu
> Description:	Ubuntu 20.04.3 LTS
> Release:	20.04
> Codename:	focal
> 

**Browser version:**
96.0.4664.110 (Official Build) (64-bit)

**Comment by Gingetsu** [comment]
@piiskop, sorry for your trouble. I checked it and this problem don't reproduce. Could you attach video with the problem?

**Comment by piiskop** [comment]
No video needed. I discovered the reason on my own. I tried to reproduce the result in another laptop and couldn't either. So I made my conclusion that it had something to do with _MATE_ that I use as a desktop environment for the particular laptop I can reproduce the issue. And I found that someone had [another issue](https://askubuntu.com/questions/1083913/what-does-ctrl-shift-e-do-while-typing-text) connected to that before. So I did `ibus-setup`, went to _Emoji_ tab and what I found was:

> Emoji annotation: <Control>period

I changed it to _C + S + e_ and now, I can subscript with _C + ._ again.
